src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import duke.commands.Command;
+import duke.data.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Duke class as main
+ */
+public class Duke {
+
+    private final Storage storage;
+    private final TaskList tasks;
+    private final Ui ui;
+
+    /**
+     * Constructor
```

Comment:
Perhaps add an empty line between the description and parameter section of the JavaDoc? I noticed the same issue in several other places too.